components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -52,37 +55,38 @@ function InputBox () {
   }
 
   return (
-    <div className={styles.container}>
-      <form className={styles.form}>
-        <div className={styles.textareaBox}>
-          <textarea
-            className={styles.textarea}
-            placeholder={getLocale('placeholderLabel')}
-            onChange={onInputChange}
-            onKeyDown={onUserPressEnter}
-            value={inputText}
-            autoFocus
-          />
-          <div className={classnames({
-            [styles.counterText]: true,
-            [styles.counterTextVisible]: isCharLimitApproaching,
-            [styles.counterTextError]: isCharLimitExceeded
-          })}>
-            {`${inputText.length} / ${MAX_INPUT_CHAR}`}
-          </div>
+    <form className={styles.form}>
+      <div className={styles.growWrap}>
+        <textarea
+          className={styles.textarea}
+          placeholder={getLocale('placeholderLabel')}
+          onChange={onInputChange}
+          onKeyDown={onUserPressEnter}
+          value={inputText}
+          autoFocus
+          rows={1}
+        />
+      </div>
+      {isCharLimitApproaching && (
+        <div className={classnames({
+          [styles.counterText]: true,
+          [styles.counterTextVisible]: isCharLimitApproaching,
+          [styles.counterTextError]: isCharLimitExceeded
+        })}>
+          {`${inputText.length} / ${MAX_INPUT_CHAR}`}
         </div>
-        <div>
-          <button
-            className={styles.buttonSend}
-            onClick={handleSubmit}
-            disabled={context.shouldDisableUserInput}
-            title={getLocale('sendChatButtonLabel')}
+      )}
+      <div className={styles.actionsContainer}>
```

Comment:
nit: do we need the suffix `Container` on `actionsContainer`? Everything is a container when it comes to the DOM 😄